src/components/expression-keypad.js

```diff
@@ -4,10 +4,10 @@
 
 const React = require('react');
 const PropTypes = require('prop-types');
-const {connect} = require('react-redux');
-const {StyleSheet} = require('aphrodite');
+const { connect } = require('react-redux');
```

Comment:
Could you do me a huge favor and pull the prettier changes out into a different PR?